site/public/js/rainbow-customization.js

```diff
@@ -326,13 +420,16 @@ function buildJSON() {
         'section' : getSection(),
         'gradeables' : getGradeableBuckets(),
         'messages' : getMessages(),
+        'plagiarism' : getPlagiarism(),
     };
 
     ret = JSON.stringify(ret);
-
     return ret;
 }
 
+
+
+
```

Comment:
These whitespace changes are seemingly irrelevant and should be reverted.  The same comment applies to the other extraneous whitespace changes throughout this file.